Monika After Story/game/script-farewells.rpy

```diff
@@ -345,6 +345,51 @@ label bye_prompt_sleep:
     $ persistent._mas_greeting_type = store.mas_greetings.TYPE_SLEEP
     return 'quit'
 
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_prompt_to_games",
+            unlocked=True,
+            prompt="I'll play other games.",
```

Comment:
This could be worded to sound more like a farewell.

I'd suggest replacing "I'll" with "I'm going to".